### DISCUSSION TITLE
Custom Item stack number

### DISCUSSION THREAD

**Discussion by PanetiX** [Zrips/CMI#7213] [Feature Requests]
Is it possible to add a function to the custom recipes so that you can specify that only a maximum of 16 pieces can be set as a stack?

For Example: I want to make a custom bread which can be only stacked with max 16 breads each stack.

**Comment by CloudeLecaw** [comment]
you can give them away with a stack size of 16 but players would be able to stack them to 64 because thats the limit of bread. Well you may need one of the item stacker plugins here for this job.
here a suggestion for one that could work with custom item stacks of modified items: https://dev.bukkit.org/projects/stackableitems
(yeah i know of the middlemouse issues in gm1 with this plugin and when you try to split stacks in gm1 with it ;) ...)

well finally an itemstacker feature for cmi itself would be a nice to have :)

**Comment by PanetiX** [comment]
is there a chance that the itemstacker feature will be added to cmi itself? :)
i would wait for that except to install another plugin :D

**Reply by CloudeLecaw** [reply]
i hope on day it will be added <3
